locale/en/blog/vulnerability/jan-2018-spectre-meltdown

```diff
@@ -0,0 +1,24 @@
+---
+date: 2018-01-08T17:30:00.617Z
+category: vulnerability
+title: Meltdown and Spectre - Impact On Node.js
+slug: jan-2018-spectre-meltdown
+layout: blog-post.hbs
+author: Michael Dawson
+---
+
+# Summary
+
+Project zero has recently announced some new attacks that have received a lot of attention: https://googleprojectzero.blogspot.ca/2018/01/reading-privileged-memory-with-side.html.
```

Comment:
nit: would you please add line breaks. Makes reviewing easier.

Reply:
That makes it more work to paste into the email list page as I then have to go and remove them all.  I can do it but means more work on the other end.